### ISSUE TITLE
Front page map: small states + tooltip

### ISSUE THREAD

**Issue by gregallensworth** [PlanScore/PlanScore#78]
On the front page map:

- [x] see about callout labels for the tiny New England states. I think Highcharts has a "small US with data labels" demo which illustrates this as a built-in capability.

- [x] the tooltips, can they center on the state instead of following the mouse? not sure if Highcharts tooltips can do that

**Comment by gregallensworth** [comment]
New England States:
Switched to the Highcharts "us-small" dataset, which includes breakout boxes for the small New England states. Note that the squares are part of their geodata (e.g. Rhode Island is a multipolygon with the state as one ring and a square as the second ring) so we don't have options for adding more spacing between that "bar" and the US mainland.

**Comment by gregallensworth** [comment]
Tooltips:
No option exists for anchoring the tooltips at the center of their object, only for anchoring them to a calculable position based on the chart size and the tooltip's width & height. As such, I placed the tooltip at the bottom-center of the map.

This created a new problem: the "click for more info" text causes this tooltip to be wide enough, to collide with the custom legend overlay. So, I shortened it to just the state name *and* changed the cursor to a pointer which is a fairly universal sign of "this can be clicked"
